app/trans/http/handler.go

```diff
@@ -0,0 +1,31 @@
+package http
+
+import (
+	"net/http"
+
+	"github.com/go-devs-ua/octagon/app/ent"
+)
+
+type User = ent.User
+
+// ApiHandler is User HTTP handler
+type ApiHandler struct {
+	logic UserLogic
```

Comment:
I would expect to see here "usecase UserUsecase" instead "logic UserLogic".

Reply:
I will try to make myself clear in further responses related to "handler"

Reply:
at the very end it looks not ok 
```go
usecase := usecase.NewUserUsecase
```